README.md

```diff
@@ -1,8 +1,131 @@
+ Deployed the application using Docker which is used to create a PostgreSQL instance to manage both static and dynamic data collection stored in RDBMS that can later be queried to produce reports for resource planning.
+
+**Core Java Apps** [[GitHub](https://github.com/jarviscanada/jarvis_data_eng_JjFalcon/tree/master/core_java)]:
+      
+  - Twitter App: Developed an app that utilizes the Twitter and RESTful APIs using the MVC architecture to enable a Twitter user to fetch, post, and delete tweets.  Tested using Maven's JUnit testing.
+  - JDBC App: Developed an application that demonstrates the DAO/DTO implementation of JDBC to a PostgreSQL RDBMS.  Implemented and tested basic CRUD and DML operations.
+  - Grep App: Developed an application that simulates the Unix egrep -R implementation that searches all files recursively from a given directory and outputs a file containing all the lines that match the given regex pattern.  Deployed via Docker.
+
+**Springboot App** [[GitHub](https://github.com/jarviscanada/jarvis_data_eng_JjFalcon/tree/master/springboot)]: COMING SOON
+
+**Python Data Analytics** [[GitHub](https://github.com/jarviscanada/jarvis_data_eng_JjFalcon/tree/master/python_data_anlytics)]: COMING SOON
+
+**Hadoop** [[GitHub](https://github.com/jarviscanada/jarvis_data_eng_JjFalcon/tree/master/hadoop)]: COMING SOON
+
+**Spark** [[GitHub](https://github.com/jarviscanada/jarvis_data_eng_JjFalcon/tree/master/spark)]: COMING SOON
+
+**Cloud/DevOps** [[GitHub](https://github.com/jarviscanada/jarvis_data_eng_JjFalcon/tree/master/cloud_devops)]: COMING SOON
+
+
+## Highlighted Projects
+**Coop Work Projects: AODA Converter** [[GitHub](https://github.com/f57c0n/JB-aodaExcel)]: Developed an application program that takes in an Excel file as input.  The program reads and convert the file to be AODA [Accessibility for Ontarians with Disabilities Act] compliant and saves it to a new Excel file. 
+
+
+**Coop Work Projects:  SQL Loader** [[GitHub](https://github.com/f57c0n/JB-sqlLoader)]: Developed an application program that takes in a CSV file as input.  The program reads and analyze the file to detect column data types and remove invalid entries, and provide DML options to review the file and a commit option to create and insert data into a remote database.  It also outputs a file with rows containing invalid entries.
+
+
+
+## Professional Experiences
+
+**Data Engineer, Jarvis (MAR2021-present)**: 
+
+ Implemented multiple progressive projects in a scrum environment following the Software Development Life Cycle using the agile methodology. 
+
+ Exposure includes architecture design, implementation via bash scripting and java using IntelliJ with Maven build automation, and JUnit testing. 
+
+ Project details are mentioned under the Jarvis Project Listing above. 
```

Comment:
are you able to keep everything in one line in case the web app displays this incorrectly